### PR TITLE
[GOVCMSD10-878] PHPUnit test for UID 1 access to GovCMS system and Drupal status reports

### PR DESCRIPTION
**Pull request by Tara-Wij**
Implement PHPUnit Test ensuring that the UID 1 user can access both the GovCMS system report and the Drupal status report.

### CODE REVIEW

File: phpunit/tests/GovCMS/Functional/Baseline/UID1ReportAccessTest.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        // Check that UID 1 can access the GovCMS system report page.
+        $this->drupalGet('admin/reports/system-report');
+        $this->assertSession()->statusCodeEquals(200);
+
+        // Verify that UID 1 can access the standard Drupal status page.
+        $this->drupalGet('admin/reports/status');
+        $this->assertSession()->statusCodeEquals(200);
+
+        // Log out to clean up the test environment.
+        $this->drupalLogout();
```
Comment (jackwrfuller) [inline phpunit/tests/GovCMS/Functional/Baseline/UID1ReportAccessTest.php:43]: i don't think it is necessary to log out of a functional test case

File: phpunit/tests/GovCMS/Functional/Baseline/UID1ReportAccessTest.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+     * {@inheritdoc}
+     */
+    protected $defaultTheme = 'stark';
+
+    /**
+     * Tests that UID 1 has access to the GovCMS system report and Drupal status report.
+     */
+    public function testUid1Access(): void {
+        // Log out the current user to ensure a clean state.
+        $this->drupalLogout();
```
Comment (jackwrfuller) [inline phpunit/tests/GovCMS/Functional/Baseline/UID1ReportAccessTest.php:29]: I don't believe it is necessary to log out before logging in as another user in a Functional drupal test.

**Review by pandaskii — APPROVED** [review]

File: phpunit/tests/GovCMS/Functional/Baseline/UID1ReportAccessTest.php [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        $this->assertSession()->statusCodeEquals(200);
+
+        // Verify that UID 1 can access the standard Drupal status page.
+        $this->drupalGet('admin/reports/status');
+        $this->assertSession()->statusCodeEquals(200);
+
+        // Log out to clean up the test environment.
+        $this->drupalLogout();
+    }
+}
```
Comment (jackwrfuller) [inline phpunit/tests/GovCMS/Functional/Baseline/UID1ReportAccessTest.php:45]: I think we need a test to cover the negative path, i.e `testNonUid1Access()`
